src/main/java/uk/nhs/digital/nhsconnect/lab/results/mesh/message/OutboundMeshMessage.java

```diff
@@ -4,26 +4,23 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 public interface OutboundMeshMessage {
-    String getHaTradingPartnerCode();
+    String getRecipient();
     WorkflowId getWorkflowId();
     String getContent();
     String getMessageSentTimestamp();
     OutboundMeshMessage setMessageSentTimestamp(String timestamp);
-    String getOperationId();
 
     @JsonCreator
     static OutboundMeshMessage create(
-            @JsonProperty(value = "haTradingPartnerCode") String haTradingPartnerCode,
+            @JsonProperty(value = "recipient") String haTradingPartnerCode,
```

Comment:
From what I can see from the rest of the code, we're not using haTradingPartnerCode, but instead we're using NHS code as the recipient
```suggestion
            @JsonProperty(value = "recipient") String nhsCode,
```